packages/block-editor/src/components/colors/use-colors.js

```diff
@@ -122,7 +123,8 @@ export default function __experimentalUseColors(
 	} = {
 		panelTitle: __( 'Color Settings' ),
 	},
-	deps = []
+	deps = [],
+	targetRef
```

Comment:
The dependencies should always be the last argument. The target ref should be part of the options object.

Reply:
Which is the second parameter?

Reply:
Yes